exporting/README.md

```diff
@@ -49,4 +49,269 @@ send charts matching = system.active_processes
+    even in that case they can occasionally synchronize their batches for a moment.
+
+## configuration
+
+In `/etc/netdata/exporting.conf` you should have something like this:
+
+```conf
+[exporting:global]
+enabled = yes
+send configured labels = no
+send automatic labels = no
+update every = 10
+
+[prometheus:exporter]
+send charts matching = system.processes
+
+[graphite:my_instance_1]
+    enabled = yes
+    destination = localhost:2003
+    data source = average
+    prefix = Netdata
+    hostname = my-name
+    update every = 10
+    buffer on failures = 10
+    timeout ms = 20000
+    send charts matching = *
+    send hosts matching = localhost *
+    send names instead of ids = yes
+
+[json:my_instance2]
+enabled = yes
+destination = localhost:5448
+data source = as collected
+update every = 2
+send charts matching = system.active_processes
+
+[opentsdb:my_instance3]
+enabled = yes
+destination = localhost:4242
+data source = sum
+update every = 10
+send charts matching = system.cpu
+
+[opentsdb:http:my_instance4]
+enabled = yes
+destination = localhost:4243
+data source = average
+update every = 3
+send charts matching = system.active_processes
+```
+
+Sections:
+- `[exporting:global]` is a section where you can set your defaults for all exporting connectors
+- `[prometheus:exporter]` defines settings for Prometheus exporter API queries (e.g.:
+  `http://your.netdata.ip:19999/api/v1/allmetrics?format=prometheus&help=yes&source=as-collected`).
+- `[<type>:<name>]` keeps settings for a particular exporting connector instance, where:
+  - type selects the exporting connector type: graphite | opentsdb:telnet | opentsdb:http | opentsdb:https |
+    prometheus_remote_write | json | kinesis | pubsub | mongodb
+  - name can be arbitrary instance name you chose.
+
+Options:
+-   `enabled = yes | no`, enables or disables an exporting connector instance
+
+-   `destination = host1 host2 host3 ...`, accepts **a space separated list** of hostnames, IPs (IPv4 and IPv6) and
+     ports to connect to. Netdata will use the **first available** to send the metrics.
+
+     The format of each item in this list, is: `[PROTOCOL:]IP[:PORT]`.
+
+     `PROTOCOL` can be `udp` or `tcp`. `tcp` is the default and only supported by the current exporting engine.
+
+     `IP` can be `XX.XX.XX.XX` (IPv4), or `[XX:XX...XX:XX]` (IPv6). For IPv6 you can to enclose the IP in `[]` to
+     separate it from the port.
+
+     `PORT` can be a number of a service name. If omitted, the default port for the exporting connector will be used
+     (graphite = 2003, opentsdb = 4242).
+
+     Example IPv4:
+
+```conf
+   destination = 10.11.14.2:4242 10.11.14.3:4242 10.11.14.4:4242
+```
+
+   Example IPv6 and IPv4 together:
+
+```conf
+   destination = [ffff:...:0001]:2003 10.11.12.1:2003
+```
+
+   When multiple servers are defined, Netdata will try the next one when the first one fails. This allows you to
```

Comment:
@joelhans should we use `previous one` instead `first one` here?

Reply:
I think you are right. I've changed it.